src/app/index.tsx

```diff
@@ -68,6 +69,10 @@ export function App() {
     dispatch(maintenanceActions.fetchMaintenance());
   }, [dispatch]);
 
+  const { data } = useUsersQuery();
+
+  useEffect(() => console.log(data), [data]);
+
```

Comment:
```suggestion
```

Reply:
@ShynRou let's comment this line out instead of removing completely, and add a note that it can be uncommented for testing purposes

Reply:
Will remove. Forgot to remove the testing code.